app/src/main/java/com/github/factotum_sdp/factotum/placeholder/UsersPlaceHolder.kt

```diff
@@ -57,28 +57,26 @@ object UsersPlaceHolder {
     fun init(dataSource: FirebaseDatabase, auth: FirebaseAuth) {
         this.dataSource = dataSource
         this.auth = auth
-        users.add(USER1)
-        users.add(USER2)
-        users.add(USER3)
-        users.add(USER_BOSS)
-        users.add(USER_COURIER)
-        users.add(USER_CLIENT)
     }
 
     /**
-     * Populates the database with users.
+     * Populates the database with user.
```

Comment:
I think it's cleaner to user suspendCoroutine to do that type of stuff, like I modified what you did before on my branch like this :

```kotlin
 suspend fun addUserToDb(user: User) = suspendCoroutine { continuation ->
        dataSource.getReference(DISPATCH_DB_PATH).push().setValue(user)
            .addOnSuccessListener {
                continuation.resume(Unit)
            }
            .addOnFailureListener { exception ->
                continuation.resumeWithException(exception)
            }
    }
```
    
Maybe look into it?